devnet/compose.py

```diff
@@ -16,7 +16,16 @@
 from typing import TypedDict, Any
 
 DEFAULT_CHAINWEB_NODE_IMAGE = "ghcr.io/kadena-io/evm-devnet-chainweb-node:latest"
-DEFAULT_EVM_IMAGE = "ghcr.io/kadena-io/kadena-reth:sha-eff370d"
+DEFAULT_MINING_CLIENT_IMAGE = "ghcr.io/kadena-io/chainweb-mining-client:latest"
+
+# This does not yet include any precompiles.
+DEFAULT_GETH_IMAGE = "ethereum/client-go"
```

Comment:
Might also be worth noting symmetrically that it doesn't do non-monotonic updates and probably doesn't refresh payloads (for now :D)